### DISCUSSION TITLE
resolve mdns .local addresses in ice

### DISCUSSION THREAD

**Discussion by viordash** [baresip/baresip#2137] [General]
Do you have any plans to implement mdns in baresip?
You probably already took this need in this [commit](https://github.com/baresip/baresip-webrtc/commit/51084d877b24c99b060aa0eefc6944f374c0ae95).
Thanks

**Comment by viordash** [comment]
FYI.
the issue with mDNS is related to the browser's local IP strategies for WebRTC (it requires some flag manipulation to disable them, i.e. [chrome](https://support.google.com/chrome/thread/75976619/anonymize-local-ips-exposed-by-webrtc-chrome-86?hl=en)).
In my application I applied this solution, if the client is from the local network, set the ice_server value to empty.

**Comment by paresy** [comment]
FYI: There seems an open Branch here: https://github.com/baresip/re/tree/icesdp_getaddr. Looks like @sreimers is/was working on this. If i can help testing i'd love to do so. (At the moment the patch is producing SIGBUS crashes for me)

**Comment by sreimers** [comment]
Yes, I'm working on mdns .local address support, the branch was a short proof of concept and is not ready yet. Will open a PR next week. So maybe it can be merged before next release.

**Comment by sreimers** [comment]
Is now supported by re main [6b8ec6901209f61d20c4bb0894586fd63e5a0293](https://github.com/baresip/re/commit/6b8ec6901209f61d20c4bb0894586fd63e5a0293)

And will be part of the next release (v3.5.0)

**Reply by paresy** [reply]
Thanks a lot!
